ArcGISRuntimeSDKQt_CppSamples/Geometry/CreateAndEditGeometries/CreateAndEditGeometries.cpp

```diff
@@ -216,6 +256,19 @@ void CreateAndEditGeometries::deleteSelectedElement()
   emit canUndoOrRedoChanged();
 }
 
+void CreateAndEditGeometries::setScaleMode(ScaleMode scaleMode)
+{
+  GeometryEditorScaleMode toolScaleMode = (scaleMode == ScaleMode::UniformScaleMode) ? GeometryEditorScaleMode::Uniform : GeometryEditorScaleMode::Stretch;
```

Comment:
Might as well make it const.
```suggestion
  const GeometryEditorScaleMode toolScaleMode = (scaleMode == ScaleMode::UniformScaleMode) ? GeometryEditorScaleMode::Uniform : GeometryEditorScaleMode::Stretch;
```